### PR TITLE
configure test/oauth-server service via yaml file

### PR DESCRIPTION
**Pull request by jkralik**
Co-authored-by: Trishia Yun <trishia1298@imsoftlab.com>

### CODE REVIEW

File: test/oauth-server/config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+apis:
+  http:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+oauthSigner:
+  idTokenKeyFile: "/secrets/private/idToken.key"
```
Comment (ondrejtomcik) [inline test/oauth-server/config.yaml:12]: Why are you signing tokens with different certs?

File: test/oauth-server/config.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  accessTokenKeyFile: "/secrets/private/accessToken.key"
+  domain: ""
```
Comment (ondrejtomcik) [inline test/oauth-server/config.yaml:14]: What is this domain used for?

File: test/oauth-server/config.yaml (hunk 1 above)
Comment (jkralik) [reply test/oauth-server/config.yaml:12]: becase idToken must by sigend by RSA (ui library doesn't supports ECDSA)
and accessToken we signed by ECDSA (is better and smaller)

File: test/oauth-server/config.yaml (hunk 2 above)
Comment (jkralik) [reply test/oauth-server/config.yaml:14]: It used as:

- issuer for idToken and accessToken
- audience for accessToken

File: test/oauth-server/config.yaml (hunk 1 above)
Comment (ondrejtomcik) [reply test/oauth-server/config.yaml:12]: Why exactly is ECDSA better if the oauth-server is used only in tests and bundle (example for development)? Why we make deployment and configuration more complex?

File: test/oauth-server/config.yaml (hunk 2 above)
Comment (ondrejtomcik) [reply test/oauth-server/config.yaml:14]: @Trishia please document what exactly is it used for.

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**